site/quickstart.md

```diff
@@ -1,159 +1,171 @@
-
-* Optionally, create a domain namespace if you want to persist the domain home in a PV:
 ```
-$ kubectl create namespace domain1-ns
+$ kubectl log -n sample-weblogic-operator-ns -c weblogic-operator deployments/weblogic-operator
 ```
-* Create the Kubernetes secrets for the Administration Server boot credentials by invoking the [`create-weblogic-credentials` script](https://github.com/oracle/weblogic-kubernetes-operator/blob/develop/kubernetes/samples/scripts/create-weblogic-domain/create-weblogic-credentials.sh).
 
-* Create a PV & PVC for the domain:
-  * Find the `create_pv_pvc.sh script` and YAML files you'll need to edit to create the PV and PVC, in the https://github.com/oracle/weblogic-kubernetes-operator/tree/develop/kubernetes/samples/scripts/create-weblogic-domain-pv-pvc directory.
-* Edit the operator YAML file to add the domain namespace, then do a `helm upgrade`.
-* Create the Docker image for the domain home in the image or use the WebLogic binary image.
-  * Run WLST to create the domain in PV (remember to apply the patch).
+## 4. Prepare your environment for a domain.
 
-## 6. Create a domain.
+a.  Create a namespace that can host one or more domains:
 
-* Edit the domain YAML file (can the defaults be used?).
-* Create the domain home for the domain.
-  * For a domain home on a PV, first pull the WebLogic 12.2.1.3 install image into a local repository:
+```
+$ kubectl create namespace sample-domains-ns1
+```
+b.	Use `helm` to configure the operator to manage domains in this namespace:
 
-```  
-$ docker pull store/oracle/weblogic:12.2.1.3-dev
 ```
- * For reference, see the [domain home on PV README](https://github.com/oracle/weblogic-kubernetes-operator/blob/develop/kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/README.md).
- * The `create-domain.sh` will:
-   * Create a directory for the generated Kubernetes YAML files for this domain. The pathname is `/path/to/weblogic-operator-output-directory/weblogic-domains/`.
-   * Create a Kubernetes job that will start up a utility WebLogic Server container and run offline WLST scripts, or WebLogic Deploy Tool (WDT) scripts, to create the domain on the shared storage.
-   * Run and wait for the job to finish.
-   * Create a Kubernetes domain resource YAML file, `domain-custom-resource.yaml`, in the directory that is created above. You can use this YAML file to create the Kubernetes resource using the `kubectl create -f` or `kubectl apply -f` command:
+$ helm upgrade \
+  --reuse-values \
+  --set "domainNamespaces={sample-domains-ns1}" \
+  --wait \
+  sample-weblogic-operator \
+  kubernetes/charts/weblogic-operator
 
 ```
-          ./create-domain.sh
-          -i create-domain-inputs.yaml
-          -o /path/to/output-directory
+c.  Configure Traefik to manage Ingresses created in this namespace:
+```
+$ helm upgrade \
+  --reuse-values \
+  --set something about namespaces
+```
+d.  Configure the operator to manage the domains created in this namespace:
```

Comment:
This is a repeat of (b)

Reply:
I definitely want to show the exact commands (for everything). That's the beauty of the quick start. Everything should be included in one place, very specific and easy to follow.

Reply:
I removed the redundant step 4d.